discordkit/utils/command.ts

```
@@ -0,0 +1,267 @@
+              }
+
+              // Handle slash command options
+              let commandArgs: T = args as T;
+              if (
+                message instanceof CommandInteraction &&
+                message.data.options
+              ) {
+                commandArgs = message.data.options.reduce(
+                  (acc, option) => {
+                    if ("value" in option) {
+                      return { ...acc, [option.name]: option.value };
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)